prow/cluster/jobs/istio/cni/istio.cni.master.yaml

```diff
@@ -0,0 +1,229 @@
+# THIS FILE IS AUTOGENERATED. See prow/config/README.md
+postsubmits:
+  istio/cni:
+  - annotations:
+      testgrid-alert-email: istio-oncall@googlegroups.com
+      testgrid-dashboards: cni-postsubmits-master
+      testgrid-num-failures-to-alert: "1"
+    branches:
+    - ^master$
```

Comment:
Do we need this to be constrained to the master branch, or should this job config apply to all branches?

Reply:
we could make it all branches, the problem is eventually we may add some incompatible changes and need to fork them off. Given cni tests don't have as much churn as Istio/Istio ones, doing all branches is probably pretty safe though